lib/utils/telemetry/generatePayload.js

```diff
@@ -298,6 +299,17 @@ module.exports = ({
     payload.config = getServiceConfig({ configuration, options, variableSources });
     payload.isConfigValid = getConfigurationValidationResult(configuration);
     payload.dashboard.orgUid = serverless && serverless.service.orgUid;
+
+    if (command === 'deploy' && isAwsProvider) {
+      const serviceName = configuration && configuration.service.name;
```

Comment:
I think `condifugration` cannot be falsy here. 

Also service name, can be set directly at `service` I think (e.g. in case where `configurationInput` is passed)

Reply:
changed